robottelo/cli/host.py

```diff
@@ -397,7 +397,7 @@ def subscription_attach(cls, options=None):
 
         Options::
 
-            --host HOST_NAME                  Name to search by
+            --host HOST_NAME    Name to search by
```

Comment:
Is that intentional?

Reply:
It might have slipped. I will change that.